src/plugins/intel_cpu/tests/functional/subgraph_tests/src/custom_op_string.cpp

```diff
@@ -0,0 +1,250 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+//      -----------------            ----------------
+//     |    Parameter    |          |    Constant    |
+//      -----------------            ----------------
+//         |  string  |                 | string |
+//   -----------    -----------------------    -----------
+//  | Extension |  |        Extension      |  | Extension |
+//   -----------    -----------------------    -----------
+//      | u8            | string       | string     | u8
+//   --------      -----------      --------      --------
+//  | Result |    | Extension |    | Result |    | Result |
+//   --------      -----------      --------      --------
+//                      | u8
+//               ---------------
+//              | Bitwise (CPU) |
+//               ---------------
+//                      | u8
+//                  --------
+//                 | Result |
+//                  --------
+
+#include <common_test_utils/ov_tensor_utils.hpp>
+#include <openvino/op/op.hpp>
+#include <shared_test_classes/base/ov_subgraph.hpp>
+#include "test_utils/cpu_test_utils.hpp"
+
+using namespace CPUTestUtils;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
```

Comment:
NIT. For Subgraph tests usually we use `SubgraphTestsDefinitions` namespace.

Reply:
Done